gradle.properties

```diff
@@ -4,25 +4,24 @@ pluginGroup = intellij
 pluginName = intellij-buf
 pluginRepositoryUrl = https://github.com/bufbuild/intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.5.1
+pluginVersion = 0.6.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 242
+pluginSinceBuild = 251
 pluginUntilBuild = 253.*
-pluginUntilVerifyBuild = 243.*
 
 # IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
 platformType = IU
-platformVersion = 2024.2
+platformVersion = 2025.1
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 platformPlugins = io.kanro.idea.plugin.protobuf:2.0.2
 # Example: platformBundledPlugins = com.intellij.java
-platformBundledPlugins = idea.plugin.protoeditor, org.jetbrains.plugins.yaml
+platformBundledPlugins = idea.plugin.protoeditor, org.jetbrains.plugins.yaml, com.intellij.modules.json
```

Comment:
Workaround for https://github.com/JetBrains/intellij-platform-gradle-plugin/issues/1930 which we've encountered in upgrading from 2.3.0 -> 2.5.0 of the plugin.